app/YtManager/settings.py

```diff
@@ -81,15 +81,19 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'postgres',
-        'USER': 'postgres',
-        'HOST': 'db',
-        'PORT': 5432,
+        'ENGINE': os.getenv('DATABASE_ENGINE', 'django.db.backends.postgresql'),
+        'NAME': os.getenv('DATABASE_NAME', 'ytmanager'),
+        'HOST': os.getenv('DATABASE_HOST', 'db'),
+        'USER': os.getenv('DATABASE_USERNAME', 'postgres'),
+        'PASSWORD': os.getenv('DATABASE_PASSWORD', 'postgres'),
+        'PORT': os.getenv('DATABASE_PORT', 5432)
     }
-
 }
 
+if os.getenv('DATABASE_URL', None):
+    import dj_database_url
```

Comment:
Is this dependency really necessary? [The documentation ](https://docs.djangoproject.com/en/2.1/ref/databases/#full-dsn-and-easy-connect)mentions that django has support for DSN.

I would prefer using the builtin DSN rather than adding a new dependency.

Reply:
I've never used django DSN for databases, but I'll take a look in a bit after fixing your previous comment of a prefix to Environment variables.

Reply:
I noticed that the DSN section is under 'Oracle notes' section, so it may only work in Oracle.

Reply:
Possibly. I know dj_database_url is pretty universal, but I'll read through the documentation and see if it'll work in this case. Hopefully someone else knows too, but I didn't see that brought up in #6

Reply:
We'll leave dj_database_url if that works universally.

Reply:
Sounds good